repair_stock_move/models/repair_order.py

```diff
@@ -172,5 +172,7 @@ def action_repair_done(self):
         if self.stock_move_ids:
             # With this module this should always be the case, so this is
             # effectively overriding the method.
+            for operation in self.operations:
+                operation.write({"state": "done"})
```

Comment:
`write` is api.multi, so you can just write `self.operations.write({"state": "done"})`